dom.bs

```diff
@@ -4860,6 +4862,14 @@ null if <var>event</var>'s {{Event/type}} attribute value is "<code>load</code>"
 <a>document</a> does not have a <a for=Document>browsing context</a>; otherwise the
 <a>document</a>'s <a>relevant global object</a>.
 
+<p>A <a>document</a>'s <dfn export id=concept-document-set-the-mode>set the mode</dfn> algorithm,
```

Comment:
It might be slightly clearer to redo this as

> To set the document mode, given a document _document_ and a string _newMode_, ...

No need to add the ID for new `<dfn>`s. One will be generated.